src/OriginDetection.php

```diff
@@ -63,6 +63,10 @@ public function parseCgroupNodePath($lines)
 
     public function getCgroupInode($cgroupMountPath, $procSelfCgroupPath)
     {
+        if (!is_readable($procSelfCgroupPath)) {
+            return '';
+        }
```

Comment:
This is a TOCTOU race. It would be better to handle errors from the actual read operation.